Typing error with app.use()

### Describe the bug

Since Vue v3.2.47, this line of code:

```ts
app.use(fluent)
```
raises typing error:

```console
❯ yarn build
yarn run v1.22.19
$ vue-tsc && vite build
src/main.ts:22:5 - error TS2554: Expected 2 arguments, but got 1.

22 app.use(fluent)
       ~~~~~~~~~~~

  node_modules/@vue/runtime-core/dist/runtime-core.d.ts:78:63
    78     use<Options extends unknown[]>(plugin: Plugin_2<Options>, ...options: Options): this;
                                                                     ~~~~~~~~~~~~~~~~~~~
    Arguments for the rest parameter 'options' were not provided.


Found 1 error in src/main.ts:22
```

I reported the issue to [Vue project](https://github.com/vuejs/core/issues/7729), but Vue core developer identified that it is `fluent-vue`'s bug.

### Reproduction

https://bitbucket.org/hongquan/vue-wrong-argument-for-use

### System Info

```Shell
System:
    OS: Linux 5.19 Ubuntu 22.10 22.10 (Kinetic Kudu)
    CPU: (4) x64 Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
    Memory: 6.45 GB / 11.59 GB
    Container: Yes
    Shell: 5.9 - /usr/bin/zsh
  Binaries:
    Node: 18.14.0 - /usr/bin/node
    Yarn: 1.22.19 - /usr/bin/yarn
    npm: 9.3.1 - /usr/bin/npm
  Browsers:
    Chrome: 110.0.5481.100
    Firefox: 110.0
  npmPackages:
    vue: ^3.2.47 => 3.2.47
```


### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/fluent-vue/.github/blob/main/CODE_OF_CONDUCT.md).
- [X] Read the [Contributing Guide](https://github.com/fluent-vue/.github/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

--- a/src/index.ts
+++ b/src/index.ts
@@ -23,7 +23,7 @@ export interface FluentVue {
 
   formatWithAttrs: (key: string, value?: Record<string, FluentVariable>) => TranslationWithAttrs
 
-  install: InstallFunction<FluentVueOptions>
+  install: InstallFunction
 }
 
 /**

--- a/src/types/typesCompat.ts
+++ b/src/types/typesCompat.ts
@@ -23,4 +23,4 @@ export type Vue2 = typeof Vue2Type
 export type Vue3 = Vue3Type
 export type Vue = Vue3 | Vue2
 
-export type InstallFunction<TOptions> = (vue: unknown, options?: TOptions) => void
+export type InstallFunction = (vue: unknown) => void

--- a/src/types/typesCompat.ts
+++ b/src/types/typesCompat.ts
@@ -23,4 +23,4 @@ export type Vue2 = typeof Vue2Type
 export type Vue3 = Vue3Type
 export type Vue = Vue3 | Vue2
 
-export type InstallFunction<TOptions> = (vue: unknown, options: TOptions) => void
+export type InstallFunction<TOptions> = (vue: unknown, options?: TOptions) => void

